experiment/autobumper/bumper/bumper.go

```diff
@@ -46,6 +48,39 @@ func Call(cmd string, args ...string) error {
 	return c.Run()
 }
 
+func call(stdout, stderr io.Writer, cmd string, args ...string) error {
```

Comment:
This doesn't fix the exported `Call()`?

Reply:
No it does not.
The only usage for that `Call` function is

https://github.com/kubernetes/test-infra/blob/5798a46d058762df45bbfac6b86bfc61e7e6983b/experiment/autobumper/main.go#L89-L94

Let me fix that too.

Reply:
It's exported, though, so anyone can use it :)

Reply:
Done.